tasks/install/CentOS.yml

```diff
@@ -4,6 +4,7 @@
     name: epel
     description: EPEL YUM repo
     baseurl: "{{ haproxy_repo }}/$releasever/$basearch/"
+  when: haproxy_repo != ''
```

Comment:
This change doesn't pass ansible-lint. A better conditional is simply `when: not haproxy_repo`.